components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5854,7 +5854,9 @@ def getBody(self):
             if self.descriptor.hasLegacyUnforgeableMembers:
                 raise TypeError("Can't handle a deleter on an interface that has "
                                 "unforgeables. Figure out how that should work!")
-            set += CGProxyNamedDeleter(self.descriptor).define()
```

Comment:
This check `if id.is_string() || id.is_int()` is somewhat redundant, as it is already present inside` jsid_to_string`. If j`sid_to_string` returns `None` because the conditions are not met, we should not panic.

Reply:
✅